DiscordBot.DataAccess/EventsSheetsService.cs

```diff
@@ -391,15 +400,16 @@ private async Task<int> GetEventRowNumber(int eventKey)
 
         private ulong? ParseMessageId(IList<object> row)
         {
-            // Does row contain a messageId cell?
-            // Or has it been trimmed off the end of the row because it is empty?
-            if (row.Count >= MessageIdColumn.Index + 1)
+            // The messageId might have been trimmed off the end of the row because it was empty
+            // so check for this.
+            var rowContainsMessageId = row.Count < MessageIdColumn.Index + 1;
+            if (rowContainsMessageId)
             {
                 return null;
             }
 
-            // Is the cell empty?
-            if ((string) row[MessageIdColumn.Index] != "")
+            var messageIdCellIsEmpty = (string) row[MessageIdColumn.Index] == "";
```

Comment:
I would store `(string) row[MessageIdColumn.Index]` instead because then you can reuse it in the parse below, as I suggested earlier 🙂

Reply:
Yep, sorry about that! Changed in 577b877 🙂